README.md

```diff
@@ -174,6 +175,22 @@ for await (const file of unpack(carReader)) {
 }
 ```
 
+### `ipfs-car/unpackStream`
+
+Takes an AsyncIterable and yields files to be consumed.
+
+```js
+import fs from 'fs'
+import { unpackStream } from 'ipfs-car/unpack'
+
+const inStream = fs.createReadStream(`${process.cwd()}/output.car`)
+
+const files = []
+for await (const file of unpackStream(inStream)) {
```

Comment:
What is a file here? A `File`?

Reply:
for now UnixFs, even though not docummented (like in original unpack)